lib/schemas/global/search/apiApplicationSummary.json

```diff
@@ -1,24 +1,174 @@
-{
-  "$schema": "http://json-schema.org/draft-04/schema#",
-  "id": "http://justice.gov.uk/core/courts/search/external/apiApplicationSummary.json",
-  "description": "An Application Summary",
-  "type": "object",
-  "properties": {
-    "applicationId": {
-      "description": "The identifier of the application",
-      "$ref": "http://justice.gov.uk/core/courts/external/apiCourtsDefinitions.json#/definitions/uuid"
-    },
-    "applicationReference": {
-      "description": "The reference of the application",
-      "type": "string"
-    },
-    "applicationTitle": {
-      "description": "The title of the application",
-      "type": "string"
-    },
-    "receivedDate": {
-      "description": "The date the application was received",
-      "$ref": "http://justice.gov.uk/core/courts/external/apiCourtsDefinitions.json#/definitions/datePattern"
-    }
-  }
-}
+{
```

Comment:
It would be great only to have the changes in the diff, instead of the entire file.
What is it? end of line or maybe different indentation? 🤔

Reply:
I wish I knew what was up with these schema files! I can't save the file inside VScode without Git thinking I've replaced the whole contents!

Reply:
I think I had to paste/copy it inside sublime text and then past again into VScode,
it's okay, the only reason it that would make review easier, but it's not worth the time.